cegs_portal/search/templates/search/v1/partials/_non_targeting_reo.html

```diff
@@ -63,3 +88,27 @@
 </div>
 
 {% include 'search/v1/partials/_row_click_js.html' %}
+
+<script>
+    document.getElementById('downloadButton').addEventListener('click', function(event) {
+        event.preventDefault();
+
+        var featureName = '{{ feature.name }}';
+
+        fetch('{% url 'search:non_target_reo' feature.accession_id %}', {
+            headers: {
+                'Accept': 'text/tab-separated-values'
+            }
+        })
+        .then(response => response.blob())
+        .then(blob => {
+            const url = window.URL.createObjectURL(blob);
+            const a = document.createElement('a');
+            a.href = url;
+            a.download = featureName + "_proximal_regulatory_observations_table_data.tsv";
+            document.body.appendChild(a);
+            a.click();
+            document.body.removeChild(a);
+        });
+    });
+    </script>
```

Comment:
Since we're setting the content disposition on the server side, this stuff isn't necessary.